includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -1,21 +1,271 @@
 <?php
 
 class DesignSystemApiController extends WikiaApiController {
+	private $data = [
+		"logo" => [
+			"header" => [
+				"type" => "link-image",
+				"href" => "http://fandom.wikia.com",
+				"image" => "wds/full_fandom_logo",
+				"title" => [
+					"type" => "text",
+					"value" => "Fandom",
```

Comment:
+1 for the full name.
As for the other languages at the moment we agreed to use user language for those components, and decided that the brand elements are non-translatable, so I'm not sure that this is a use case we want to support.